domain/src/main/java/org/oppia/android/domain/exploration/ExplorationSessionTimerController.kt

```diff
@@ -0,0 +1,59 @@
+package org.oppia.android.domain.exploration
+
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.domain.oppialogger.analytics.ApplicationLifecycleListener
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Controller to decide when the session timer is started and stopped based on signals received
+ * from [ExplorationProgressListener] and [ApplicationLifecycleListener].
+ */
+@Singleton
+class ExplorationSessionTimerController @Inject constructor(
+  private val explorationActiveTimeController: ExplorationActiveTimeController
+) : ExplorationProgressListener, ApplicationLifecycleListener {
```

Comment:
Why not have the ``ExplorationActiveTimeController`` itself bind as an instance of these listeners? It seems like the extra wrapper doesn't add much benefit (unless I'm missing something).

Reply:
I refactored this and removed ExplorationSessionTimerController and its test file

Reply:
This looks done.